The procyon 0.5.32 decompiler is not compatible with JAVA 9 and above

The procyon 0.5.32 decompiler is not compatible with java 9 and above:

Reproductions steps:

1. Run AEM on Java 11
2. Install websearch console plugin and try to decompile something

Actual results:
![image](https://user-images.githubusercontent.com/18657753/74220984-4ba13580-4cb1-11ea-9383-41ebb7743db2.png)

In log I can find entries like:
`Caused by: java.lang.ClassNotFoundException: sun.misc.URLClassPath not found by com.neva.felix.webconsole.plugins.search [606]`

--- a/pom.xml
+++ b/pom.xml
@@ -247,6 +247,6 @@
         <felix.url>http://localhost:8080/system/console</felix.url>
         <felix.user>admin</felix.user>
         <felix.password>admin</felix.password>
-        <procyon.version>0.5.32</procyon.version>
+        <procyon.version>0.5.36</procyon.version>
     </properties>
 </project>

